### PR TITLE
Closes AgileVentures/Metplus_tracker#170

### PR DESCRIPTION
**Pull request by sandhyakoti**

### CODE REVIEW

File: app/views/companies/_subform.html.haml [hunk 1]
```diff
@@ -7,42 +7,46 @@
...
-  .col-sm-4
+  .col-sm-6
     = @address_form.email_field :email, size: 30, class: 'form-control'
+
+.form-group
   .col-sm-2.control-label
     = label :company, :phone, 'Phone'
-  .col-sm-4
-    = @address_form.phone_field :phone, size: 30, class: 'form-control',
-    placeholder: 'Eg. 1-369-374-5803 x9579 (\'1-\' prefix and extension optional)'
```
Comment (joaopapereira) [inline app/views/companies/_subform.html.haml]: Was there any reason to loose this placeholder?

File: app/views/companies/_subform.html.haml [hunk 2]
```diff
@@ -7,42 +7,46 @@
...
     %i.fa.fa-question-circle{data_toggle: 'tooltip', data_placement: 'top',
               title: 'Email where job applications will be sent'}
-  .col-sm-1.control-label
+
+.form-group              
+  .col-sm-2.control-label
     = label :company, :fax, 'Fax'
-  .col-sm-4
-    = @address_form.phone_field :fax, size: 30, class: 'form-control',
-    placeholder: 'Eg. 1-369-374-5803 x9579 (\'1-\' prefix and extension optional)'
```
Comment (joaopapereira) [inline app/views/companies/_subform.html.haml]: Was there any reason to loose this placeholder?

**Comment by joaopapereira** [comment]
The enhancements look good, but please review my comments

**Comment by patmbolger** [comment]
The placeholders should definitely be retained.

Also, it might be useful to group the "email" and "phone" fields together.  Right now, these fields are ordered as:

screen shot 2016-07-18 at 4 15 00 pm

Suggested ordering:

1. Email
2. Job Email
3. Phone
4. Fax

**Comment by sandhyakoti** [comment]
Hello,

Losing the placeholders was not intentional. Looks like the merge has been
inappropriate. I will correct it.

Meanwhile, could you please tell me what the "Title" field in the Contact
Person is used for? Is it for Mr/Ms/Mrs or does it specify the Designation
of the person in the company?

I think that field is not clear and needs to be addressed.

Thanks,
Sandhya Koti

**Comment by patmbolger** [comment]
That's a very good question.

The ‘Title’ is the job title of the company contact person.  For instance, “Human Resources Manager”, “IT System Admin”, etc.

So, perhaps the label should say 'Job Title'.

**Comment by joaopapereira** [comment]
Looks good, going to merge

**Pull request merged**